Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2008-11-21
Initial Package Version: 1.2.2
Upstream Status: unknown
Origin: Mandriva
Description: Allows it to compile with gcc-4.2.

Purpose: fix build issue, 'SigC has not been declared'.
Originally from Adam Williamson at mandriva, via Ichiro Nakai at momonga
linux, and google.

diff -Naur cdrdao-1.2.2.orig/xdao/xcdrdao.cc cdrdao-1.2.2/xdao/xcdrdao.cc
--- cdrdao-1.2.2.orig/xdao/xcdrdao.cc	2005-05-03 10:50:17.000000000 +0100
+++ cdrdao-1.2.2/xdao/xcdrdao.cc	2007-10-19 23:57:08.000000000 +0100
@@ -136,7 +136,7 @@
   installSignalHandler(SIGCHLD, signalHandler);
 
   // setup periodic GUI updates
-  Glib::signal_timeout().connect(SigC::slot(&guiUpdatePeriodic), 2000);
+  Glib::signal_timeout().connect(sigc::ptr_fun(&guiUpdatePeriodic), 2000);
 
   installSignalHandler(SIGPIPE, SIG_IGN);
 
